website/client-old/js/controllers/newGroupCtrl.js

```diff
@@ -0,0 +1,53 @@
+"use strict";
+
+/*
+ A controller to manage the New Group page
+ */
+
+angular.module('habitrpg')
+  .controller("NewGroupCtrl", ['$scope', '$window', 'Groups', 'Payments',
```

Comment:
I was just following precedent in that new pages seem to always get new controllers, e.g. from habitica/website/client-old/js/app.js:129 onwards, all the "social" sub-states have their own template pages and their own controller.
The original page needed to be split into two because its 'create group' sub-section is now being linked into from two locations (a 'static' front page where the user may be outside the login wall), and the internal group plans page.